File: input3.c
#include<stdio.h>
#include<stdlib.h>

void banner(){
	printf("======================");puts("");
	printf("--------GuGuDan-------");puts("");
	printf("======================");puts("");
}
void main(){
	banner();
	for(int x=1;x<10;x++)
	{
		printf("=== %d Dan ===",x);puts("");
		for(int y=1;y<10;y++)
		{
			printf("%d * %d = %d",x, y, x*y);puts("");
		}
		printf("=============");puts("");
	}
}

File: input3.py
def banner() :
	print("======================")
	print("--------GuGuDan-------")
	print("======================")

def main() :
	banner()

	for x in range(1,10):
		print("=== %d Dan ==="%(x))
		for y in range(1,10):
			print("%d * %d = %d"%(x, y, x*y))
		print("=============")

if __name__ == "__main__" :
	main()
